docs/reference/commandline/run.md

```diff
@@ -243,9 +246,12 @@ An example of a file passed with `--env-file`
     helloWorld=true
     TEST_PASSTHROUGH=
     HOME=/root
+    123qwe=bar
+    org.spring.config=something
 
-> **Note**: Environment variables names must consist solely of letters, numbers,
-> and underscores - and cannot start with a number.
+> **Note**: Environment variables are passed as-is, see
+> [here](http://pubs.opengroup.org/onlinepubs/009695399/basedefs/xbd_chap08.html)
+> for more information about it.
```

Comment:
I would vote for dropping the note entirely? Do we really need something saying "we don't mess with the data you gave us"?
Though that technically inaccurate, as we strip leading spaces.